.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["react-app", "plugin:prettier/recommended"],
+  "extends": ["plugin:prettier/recommended"],
   "plugins": ["prettier"],
```

Comment:
I think this line is not needed if you have already have `plugin:prettier/recommended`.
Also, the rules are not needed. It's implied by the use of the recommended configuration above
```js
"rules": {
    "prettier/prettier": "error"
  }
```